src/python/pants/init/engine_initializer.py

```diff
@@ -315,25 +316,21 @@ def setup_legacy_graph_extended(
     :param build_file_imports_behavior: How to behave if a BUILD file being parsed tries to use
       import statements. Valid values: "allow", "warn", "error".
     :type build_file_imports_behavior: string
-    :param str build_root: A path to be used as the build root. If None, then default is used.
+    :param build_root: A path to be used as the build root. If None, then default is used.
```

Comment:
I'm not sure I understand why `str` is being removed from the docstring here while `Native` is kept, for example?

Reply:
those should be removed too! Good catch!